src/components/FeatureViews/Registration/RegistrationLanding.js

```diff
@@ -16,19 +17,31 @@ import Loading from "../../Loading";
 import CourseList from "./CourseList";
 import TutoringList from "./TutoringList";
 import RegistrationActions from "./RegistrationActions";
+import * as adminActions from "../../../actions/adminActions";
+import { bindActionCreators } from "redux";
 
-const NUM_GRADES = 13;
 
-const gradeOptions = Array(NUM_GRADES).map((_, gradeNum) => ({
-    "label": `${gradeNum + 1}`,
-    "value": gradeNum + 1,
-}));
+const NUM_GRADES = 13;
 
 const RegistrationLanding = () => {
-    const courses = useSelector(({"Course": {NewCourseList}}) => NewCourseList);
-    const instructors = useSelector(({"Users": {InstructorList}}) => InstructorList);
+    const courses = useSelector(({ "Course": { NewCourseList } }) => NewCourseList);
+    const instructors = useSelector(({ "Users": { InstructorList } }) => InstructorList);
+    const categories = useSelector(({ "Course": { CourseCategories } }) => CourseCategories)
+
     const courseStatus = hooks.useCourse();
-    hooks.useInstructor();
+    hooks.useInstructor()
+
+    const dispatch = useDispatch();
+    const api = useMemo(
+        () => ({
+            ...bindActionCreators(adminActions, dispatch),
+        }),
+        [dispatch]
+    );
+
+    useEffect(() => {
+        api.fetchCategories();
+    }, [api]);
```

Comment:
Replace this with the `useCategory` hook (imported as `hooks`).

Reply:
Please explain. 
<img width="735" alt="Screen Shot 2020-03-09 at 10 13 52 AM" src="https://user-images.githubusercontent.com/40410706/76239369-b0877700-61ee-11ea-998c-94b9692cf38e.png">
error:
<img width="1168" alt="Screen Shot 2020-03-09 at 10 14 27 AM" src="https://user-images.githubusercontent.com/40410706/76239421-c39a4700-61ee-11ea-98cf-0ca548f6f3ad.png">